atomicapp/providers/openshift.py

```diff
@@ -176,6 +198,81 @@ def deploy(self):
                     continue
                 self.oc.deploy(url, artifact)
 
+    def undeploy(self):
```

Comment:
Maybe add description and also note the difference between deleting a replication controller or deployment config when using the cli vs when using the API. Basically when using the cli it will clean up the pods/repication controllers automatically when you delete a DC. For deleting an rc it will automatically delete the pods. When using the API we have to do this stuff manually.